plugins/experimental/remap_purge/remap_purge.c

```diff
@@ -143,18 +144,17 @@ on_http_cache_lookup_complete(TSHttpTxn txnp, TSCont contp, PurgeInstance *purge
 }
 
 /* Before we can send the response, we want to modify it to a "200 OK" again,
-   and produce some reasonabel body output. */
+   and produce some reasonable body output. */
 static int
 on_send_response_header(TSHttpTxn txnp, TSCont contp, PurgeInstance *purge)
 {
   TSMBuffer bufp;
   TSMLoc hdr_loc;
 
-  TSDebug(PLUGIN_NAME, "Fixing up the response on the succseful PURGE");
+  TSDebug(PLUGIN_NAME, "Fixing up the response on the successful PURGE");
   if (TS_SUCCESS == TSHttpTxnClientRespGet(txnp, &bufp, &hdr_loc)) {
     char response[1024];
-    int len = snprintf(response, sizeof(response) - 1, "PURGED %s\r\n\r\n", purge->id);
-    ;
+    int len = snprintf(response, sizeof(response), "PURGED %s\r\n\r\n", purge->id);
```

Comment:
``len`` is what ``snprintf`` would have printed. Should be fine in practice, but in principle could be > ``strlen(response)``.